src/Compilers/Core/Portable/Symbols/ISymbolInternal.cs

```diff
@@ -2,10 +2,24 @@
 
 #nullable enable
 
+using System.Collections.Immutable;
+
 namespace Microsoft.CodeAnalysis.Symbols
 {
-    internal interface ISymbolInternal : ISymbol
+    internal interface ISymbolInternal
     {
```

Comment:
Please add a lot of documentation to this declaration to help the reader understand how this fits into the scheme of things.  I'd rather not have to guess.